quotes/info.json

```diff
@@ -4,5 +4,6 @@
 	"install_msg": "Thanks for installing. After the cog is loaded, use `[p]help Quotes` to see the available settings and commands.",
 	"short": "Save quotes and read them later",
 	"tags": ["quotes", "quote"],
-	"type": "COG"
+	"type": "COG",
+	"end_user_data_statement": "This cog does not persistently store data or metadata about users."
```

Comment:
This cog may store information about a user for the purposes of relaying quotes. This data cannot be tracked for deletion and this cog will not remove content if requested.